scripts/nuke.py

```diff
@@ -19,6 +20,7 @@
                         archive path for a job to kill and nuke
   --stale               attempt to find and nuke 'stale' machines
                         (e.g. locked by jobs that are no longer running)
+  --dry-run             Don't actually nuke anything
```

Comment:
Ok, can we change the help text to mention that then? "Don't actually nuke anything, but output the list of targets to be nuked". Maybe?